crawler/plugins_manager.py

```diff
@@ -137,16 +137,43 @@ def reload_host_crawl_plugins(
     # features to be plugins.
 
 
-def get_container_crawl_plugins():
+def get_container_crawl_plugins(
+    features=[
+        'package',
+        'os',
+        'process',
+        'file',
+        'config']):
```

Comment:
It was to make some tests work. The issue is the following. Some tests are running `snapshot_container` directly which gets a list of features as an argument. So, when running, we need to load the plugins related to those features when running that function.